mteb/tasks/aggregated_tasks/STS17MultilingualVisualSTSEng.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTask
+from mteb.abstasks.aggregated_task import AbsTaskAggregate, AggregateTaskMetadata
+from mteb.tasks.Image.VisualSTS import STS17MultilingualVisualSTS
+
+task_list_sts17: list[AbsTask] = [
+    STS17MultilingualVisualSTS().filter_languages(
+        languages=["eng"], hf_subsets=["en-en"]
+    )
+]
+
+
+class STS17MultilingualVisualSTSEng(AbsTaskAggregate):
```

Comment:
I don't thik there is a need to introduce the filtering here.

I think simply in the benchmark def:

```
...
tasks = MTEBTasks(...
  mteb.get_tasks("STS17MultilingualVisualSTS", langauges = ["eng"], hf_subsets=["en-en"])
...
)
```

e.g. see `MTEB(eng, v1)`.

Reply:
Reverted to using get_task / get_tasks.